pkg/uroot/builder/bb/bb.go

```diff
@@ -259,6 +259,11 @@ func RewritePackage(env golang.Environ, pkgPath, bbImportPath string, importer t
 		return err
 	}
 	dest := filepath.Join(buildp.Dir, ".bb")
+	// If .bb directory already exists, delete it. This will prevent stale
```

Comment:
should we instead just drop this into a os.TempDIr somewhere? 

removeall scares me. Anyway, I'll approve it, nice catch if this is what it was.

Reply:
We can't. I wish.

Go build system reasons (vendor dependencies)